apps/coa/translations/src/en/validation.json

```diff
@@ -171,11 +171,16 @@
     "privacy-check": {
       "required": "Confirm you have read the Data Protection statement"
     },
-    "document-file": {
-        "default": "Select a file to upload",
+    "file-upload": {
+        "required": "Select a file to upload",
         "unsaved": "Your file could not be uploaded – try again",
-        "maxFileSize": "Your file must be smaller than 25MB",
-        "fileType": "Your file must be a JPG, JPEG, PNG or PDF",
-        "virusCheck": "The file you have uploaded may have a virus. Try a different file"
+        "maxFileSize": "Your file must be smaller than {{maxFileSize}}",
+        "fileType": "Your file must be a {{fileType}}",
+        "virusCheck": "The file you have uploaded may have a virus. Try a different file",
+        "maxIdDocsUploads" : "You can only upload a {{maxIdDocsUploads}} files or less. Remove a file before uploading another",
+        "maxAddressDocsUploads" : "You can only upload a {{maxAddressDocsUploads}} files or less. Remove a file before uploading another",
```

Comment:
Change of text  for `maxIdDocsUploads` and `maxAddressDocsUploads`
You can only upload `a `....
You can only upload `up to `....